.rubocop_todo.yml

```diff
@@ -1,32 +1,91 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2021-04-14 06:51:37 UTC using RuboCop version 0.90.0.
+# on 2021-08-16 15:31:43 UTC using RuboCop version 1.19.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 26
-# Configuration parameters: IgnoredMethods.
+# Offense count: 7
+# Configuration parameters: AllowedMethods.
+# AllowedMethods: enums
+Lint/ConstantDefinitionInBlock:
+  Exclude:
+    - 'spec/faraday/composite_read_io_spec.rb'
+    - 'spec/faraday/options/options_spec.rb'
+    - 'spec/faraday/rack_builder_spec.rb'
+    - 'spec/faraday/request/instrumentation_spec.rb'
+
+# Offense count: 11
+# Configuration parameters: AllowComments, AllowEmptyLambdas.
+Lint/EmptyBlock:
+  Exclude:
+    - 'spec/faraday/connection_spec.rb'
+    - 'spec/faraday/rack_builder_spec.rb'
+    - 'spec/faraday/response_spec.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
```

Comment:
This and other cops added to the `.rubocop_todo.yml` support `--auto-correct`.
Is there a reason why we don't simply let Rubocop auto-correct them?

Reply:
I just wanted to stay green, while doing it, let's see if we can get this done before closing the lid for today!

Reply:
@iMacTia There! Now those are out of the file, and handled.